python/tskit/util.py

```diff
@@ -316,6 +316,38 @@ def obj_to_collapsed_html(d, name=None, open_depth=0):
         return f"{name} {d}"
 
 
+def unicode_table(title, header, rows):
```

Comment:
A quick docstring would help here. (no need for full sphinx param descriptions, etc)

Reply:
Done. Also added a docstring for the HTML method above this one.